linters/README.md

```diff
@@ -5,3 +5,5 @@ Files to configure the linters we use for each language and technology
 
 * Ruby
   * [Rubocop](/linters/ruby/.rubocop.yml)
+* JavaScript
```

Comment:
No, there is nothing major that changes in terms of syntax. And it's very easy to take the configs and just change the ones that don't apply to that version of ruby, which for very old version would be one or two cops.

Will that linter config work with ES7 and ES8?